satpy/tests/reader_tests/test_fci_l1c_nc.py

```diff
@@ -57,9 +57,135 @@
+                          "segment_height": 100,
+                          "grid_width": 5568}},
+        "hrfi": {"500m": {"start_position_row": 1,
+                          "end_position_row": 400,
+                          "segment_height": 400,
+                          "grid_width": 22272},
+                 "1km": {"start_position_row": 1,
+                         "end_position_row": 200,
+                         "grid_width": 11136,
+                         "segment_height": 200}},
+        "fci_af" : {"3km": {"start_position_row": 1,
+                          "end_position_row": 67,
+                          "segment_height": 67,
+                          "grid_width": 3712
+                          },
+                    },
+        "fci_af_vis_06" : {"3km": {"start_position_row": 1,
+                          "end_position_row": 67,
+                          "segment_height": 67,
+                          "grid_width": 3712
+                          },
+                        "1km": {"start_position_row": 1,
+                         "end_position_row": 200,
+                         "grid_width": 11136,
+                         "segment_height": 200}
+                    }
+    }
+
+_CHANS_FDHSI = {"solar": LIST_CHANNEL_SOLAR,
+                "solar_grid_type": ["1km"] * 8,
+                "terran": LIST_CHANNEL_TERRAN,
+                "terran_grid_type": ["2km"] * 8}
+
+_CHANS_HRFI = {"solar": ["vis_06", "nir_22"],
+               "solar_grid_type": ["500m"] * 2,
+               "terran": ["ir_38", "ir_105"],
+               "terran_grid_type": ["1km"] * 2}
+
+DICT_CALIBRATION = { "radiance" : {"dtype": np.float32,
+                      "value_1": 15,
+                      "value_0":9700,
+                      "attrs_dict":{"calibration":"radiance",
+                                       "units":"mW m-2 sr-1 (cm-1)-1",
+                                      "radiance_unit_conversion_coefficient": np.float32(1234.56)
+                                    },
+                    },
+
+                    "reflectance" : {"dtype": np.float32,
+                                    "attrs_dict":{"calibration":"reflectance",
+                                       "units":"%"
+                                    },
+                    },
+
+                 "counts" : {"dtype": np.uint16,
+                    "value_1": 1,
+                    "value_0": 5000,
+                    "attrs_dict":{"calibration":"counts",
+                                       "units":"count",
+                                    },
+                    },
+
+            "brightness_temperature" : {"dtype": np.float32,
+                "value_1": np.float32(209.68275),
+                "value_0": np.float32(1888.8513),
+                "attrs_dict":{"calibration":"brightness_temperature",
+                                "units":"K",
+                                      },
+                },
+}
+_test_filenames = {"fdhsi": [
```

Comment:
again, why the leading underscore? also, should this list be CONSTANT, i.e. capitalized?

Reply:
It's done I have erased what you the underscore